### DISCUSSION TITLE
Borderless and transparant

### DISCUSSION THREAD

**Discussion by septarchy** [Go1den/new-stream-ticker#27] [Feature Requests]
Hello Sir.

Thanks for making this project! 

I do have one feature request: Is it possible to make the window bordless and transparant? I want it to fade away in the background... 
That would be marvelous :)

**Comment by Go1den** [comment]
I tried implementing borderless windows in the past but it was causing some weird behavior. Transparency I'm not sure would be easy to accomplish. Most of this can be worked around in OBS fairly easily though.
